src/framework/audio/cmake/SetupFlac.cmake

```diff
@@ -32,8 +32,8 @@ if (MUE_COMPILE_USE_SYSTEM_FLAC)
     pkg_check_modules(flac REQUIRED IMPORTED_TARGET flac)
     pkg_check_modules(flacpp REQUIRED IMPORTED_TARGET flac++)
 
-    if (FLAC_FOUND)
-        message(STATUS "Found flac: ${FLAC_VERSION}")
+    if (flac_FOUND)
+        message(STATUS "Found flac: ${flac_VERSION}")
```

Comment:
BTW even with this change, I still get in the cmake configure phase:

```
CMake Warning at /nix/store/sa1yjb7xxrzcbd1bysj3r561vidfxxvb-cmake-3.29.6/share/cmake-3.29/Modules/CMakeFindDependencyMacro.cmake:76 (find_package):
  By not providing "FindOgg.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Ogg", but
  CMake did not find one.

  Could not find a package configuration file provided by "Ogg" with any of
  the following names:

    OggConfig.cmake
    ogg-config.cmake

  Add the installation prefix of "Ogg" to CMAKE_PREFIX_PATH or set "Ogg_DIR"
  to a directory containing one of the above files.  If "Ogg" provides a
  separate development package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  /nix/store/pba2k9smqpapmc6w6l2csq1h4820ak0n-flac-1.4.3-dev/lib/cmake/FLAC/flac-config.cmake:29 (find_dependency)
  src/framework/audio/cmake/SetupFlac.cmake:22 (find_package)
  src/framework/audio/CMakeLists.txt:274 (include)


CMake Warning at src/framework/audio/cmake/SetupFlac.cmake:22 (find_package):
  Found package configuration file:

    /nix/store/pba2k9smqpapmc6w6l2csq1h4820ak0n-flac-1.4.3-dev/lib/cmake/FLAC/flac-config.cmake

  but it set FLAC_FOUND to FALSE so package "FLAC" is considered to be NOT
  FOUND.  Reason given by package:

  FLAC could not be found because dependency Ogg could not be found.

Call Stack (most recent call first):
  src/framework/audio/CMakeLists.txt:274 (include)
```

And indeed I don't have `OggConfig.cmake` or alike found in `CMAKE_PREFIX_PATH`, but I do have a `pkg-config` `ogg.pc` file... I did notice that if I build `libogg` with cmake, I am capable of generating an `OggConfig.cmake`, but I wonder whether that can be avoided?

Reply:
My guess would be that this warning actually stems from line 22. The idea is that we first try to find Flac using a normal find_package, and if that doesn't succeed, we fall back to PkgConfig.
The reason that we don't use PkgConfig only, is the following: on macOS we use an archive of precompiled dependencies, which also contains Flac; to use the version from the precompiled archive rather than compiling from source, we enable `MUE_COMPILE_USE_SYSTEM_FLAC` on macOS. But because PkgConfig files contain hardcoded paths, this means that the archive with dependencies must be installed in exactly the same location on the machine where MuseScore is built as on the machine where these dependencies were built. That's of course something that cannot be guaranteed. 
If you know a way to somehow make PkgConfig files use relative paths, then we can implement that for the dependencies archive, and always use PkgConfig in SetupFlac.cmake.

Other options might be to set this `find_package` to `QUIET`, or to provide our own `FindFlac.cmake` somehow. I'm not sure how nice that would be though.

Reply:
Anyway, any further changes to this will have to happen in a new PR, because this one is just to keep `master` and `4.4.1` in sync.